src/main/java/bio/terra/service/filedata/google/firestore/FireStoreDirectoryDao.java

```diff
@@ -191,8 +191,7 @@ public Map<UUID, UUID> upsertDirectoryEntries(
         .filter(Objects::nonNull)
         .collect(Collectors.toMap(IdConflict::attemptedId, IdConflict::existingId));
   }
-  // Need to declare the record class out here and not inline in the method sine that makes spotbugs
```

Comment:
Can `IdConflict` be moved inside `upsertDirectoryEntries()` now?

Reply:
I am not 100% confident on this, so would prefer not to modify as it's not doing any harm to have it separate (but if you have a concrete suggestion I'm open to it).

Reply:
Seems OK to me. Making it local would work, but there's no great harm it leaving it where it is for now too.